samples/getting-started/DemoApp/ViewController.swift

```diff
@@ -11,6 +11,11 @@ import UIKit
 class ViewController: UIViewController {
 
     let label = UILabel()
+    let buttonForAlert = Button()
+    let buttonForAction = Button()
+    
+    let alertController = UIAlertController(title: "Alert Message", message: alertMessageShortest, preferredStyle: .alert)
+    let actionsController = UIAlertController(title: "Actions", message: nil, preferredStyle: .actionSheet)
```

Comment:
Thought it might be nice to add this to the DemoApp. But we could also revert all these changes here.

Reply:
I think we don't need to make these members of the `ViewController` though.

In fact they can be initialised only when needed (in the button handlers).
Ex:
```
private func displayAlertController() {
    let alertController = UIAlertController(...)
   alertController.addAction(...)
   ....
   self.present(alertController, animated: true)
}

// handling actions
buttonForAlert.onPress = {
   self.displayAlertController()
}
```

Reply:
I think this might also fix the issue you were facing with the alert not being displayed a second time.